### DISCUSSION TITLE
HPC-Stack installation permission on Hercules

### DISCUSSION THREAD

**Discussion by ywangwof** [ufs-community/ufs-srweather-app#919] [Q&A] [answered]
It seems that general users do not have read permission for the following HPC-Stack installation. So the module _/work2/noaa/wof/ywang/UFS/ufs-srweather-app/modulefiles/build_hercules_intel.lua_ cannot be loaded.

```
[hercules-login-1:/work2/noaa/wof/ywang/UFS/ufs-srweather-app/modulefiles]$ l /work/noaa/epic/role-epic/contrib/hercules/hpc-stack
total 80K
drwxr-s---  5 role-epic epic 16K Sep 16 18:25 intel-2022.2.1
```

Cloud someone fix the permission? Thanks.

**Comment by EdwardSnyder-NOAA** [comment, marked as answer]
Good morning @ywangwof,

Read permissions where added to the HPC-stack in question. Feel free to give it a try, and let me know if you run into any other issues.

Thanks,

Eddie Snyder | Advanced User Support (AUS) Team
